lib/utils/logDeprecation.js

```diff
@@ -165,7 +172,7 @@ module.exports.printSummary = async () => {
     ]);
     await safeMoveFile(tmpHealthStatusFilename, healthStatusFilename);
 
-    if (bufferedDeprecations.length === 1) {
+    if (bufferedDeprecations.length === 1 && !isModernStyle) {
```

Comment:
More accurate would be to add it at L80. Theoretically both modern and legacy logs could be outputted, and in this form, it'll prevent the writing of legacy logs in such case

Reply:
Good point - I've changed the approach - if you see a better solution please let me know